### ISSUE TITLE
Version 1.1 is bugged

### ISSUE THREAD

**Issue by enumag** [MrOctopus/download-asset-action#8]
I tried to use the new version but it fails with this error:

```
Error: fetch is not set. Please pass a fetch implementation as new Octokit({ request: { fetch }}). Learn more at https://github.com/octokit/octokit.js/#fetch-missing
```

From the linked documentation it seems you can either provide fetch manually or upgrade to node 18+.

**Comment by MrOctopus** [comment]
Fixing it asap

**Comment by MrOctopus** [comment]
Fixed in d66a4974cc2af592ea34524649da60634a7266e1 and updated the 1.1 tag/release with the newest commit.

**Comment by enumag** [comment]
Thank you!
